### PR TITLE
[Doc] Add Cache Select doc for v3.3

### PR DESCRIPTION
**Pull request by evelynzhaojie**
## What I'm doing:

this pr is a work on #43903

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/zh/data_source/data_cache_warmup.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+| STATUS  | ALREADY_CACHED_SIZE | WRITE_CACHE_SIZE | AVG_WRITE_CACHE_TIME | TOTAL_CACHE_USAGE |
++---------+---------------------+------------------+----------------------+-------------------+
+| SUCCESS | 15.2MB              | 754.7MB          | 88.7ms               | 29.04%            |
++---------+---------------------+------------------+----------------------+-------------------+
+1 row in set (36.56 sec)
+```
+
+返回字段说明：
+
+- `STATUS`：预热任务的执行结果。
```
Comment (EsoragotoSpirit) [inline docs/zh/data_source/data_cache_warmup.md:57]: 要写下所有可能的状态吗？

File: docs/zh/data_source/data_cache_warmup.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+| STATUS  | ALREADY_CACHED_SIZE | WRITE_CACHE_SIZE | AVG_WRITE_CACHE_TIME | TOTAL_CACHE_USAGE |
++---------+---------------------+------------------+----------------------+-------------------+
+| SUCCESS | 1.3GB               | 15.1GB           | 12.2ms               | 75.81%            |
++---------+---------------------+------------------+----------------------+-------------------+
+1 row in set (2 min 17.06 sec)
+```
+
+### verbose 模式预热
+
+默认情况下，`CACHE SELECT` 返回的指标是一个合并后的指标，您可以在 `CACHE SELECT` 末尾添加 `PROPERTIES("verbose"="true")` 获得各个 BE 上更加详细的指标。
```
Comment (EsoragotoSpirit) [inline docs/zh/data_source/data_cache_warmup.md:79]: 只能是 BE 节点吗？还是 BE 或 CN？下同

File: docs/zh/data_source/data_cache_warmup.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+3 rows in set (42.87 sec)
+```
+
+`verbose` 模式下，会返回每个 BE 的详细预热情况。同时会多返回一个指标：
+
+`AVG_READ_CACHE_TIME`：表示每一个文件在 Data Cache 里的平均查找耗时。
+
+## CACHE SELECT 定时调度
+
+CACHE SELECT 可以和 [SUBMIT TASK](../sql-reference/sql-statements/data-manipulation/SUBMIT_TASK.md) 结合，以实现周期性的预热。
```
Comment (EsoragotoSpirit) [inline docs/zh/data_source/data_cache_warmup.md:99]: 这个在 SUBMIT TASK 文档里加一下吧。

File: docs/zh/data_source/data_cache_warmup.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+参数说明：
+
+- `column_name`：要拉取的列，多列用逗号隔开。也可以不指定，使用 `*` 来拉取表中所有列。
+- `catalog_name`：远端 Catalog 名称。如果已经通过 SET CATALOG 切换到远端 Catalog 下，也可以不填。
+- `db_name`：远端数据库名称。如果已经切换到远端数据库下，也可以不填。
+- `table_name`：远端表名称。
+- `boolean_expression`: WHERE 中指定的过滤条件。
+- `PROPERTIES`：当前仅支持设置 `verbose` 属性，用于返回详细的预热指标。
+
+`CACHE_SELECT` 是一个同步过程，且一次只能对一个表进行预热。执行成功后，会返回 Cache 的指标。
```
Comment (EsoragotoSpirit) [inline docs/zh/data_source/data_cache_warmup.md:39]: 下划线应该删掉？

File: docs/en/data_source/data_cache_warmup.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
+---
+displayed_sidebar: "English"
+---
+
+# Data cache warmup
+
+Some data lake analytics scenarios have high performance requirements for queries, such as BI reports and proof of concept (PoC) performance testing. Loading remote data into local data cache can avoid the need to fetch the same data multiple times, significantly speeding up query execution and minimizing resource usage.
+
+StarRocks v3.3 introduces the Data Cache Warmup feature, which is an enhancement to [Data Cache](./data_cache.md). Data Cache is a process of passively populating the cache, in which data is written to the cache during data querying. Data Cache Warmup, however, is an active process of populating the cache. It proactively fetching the desired data from remote storage in advance.
```
Comment (EsoragotoSpirit) [inline docs/en/data_source/data_cache_warmup.md:9]: fetching --> fetches

File: docs/en/data_source/data_cache_warmup.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+- `column_name`: The columns to fetch. You can also use `*` to fetch all columns in the remote table.
+- `catalog_name`: The name of the remote catalog. If you have switched to the remote catalog using SET CATALOG, it can be left unspecified.
+- `db_name`: The name of the remote database. If you have switched to the remote database, it can be left unspecified.
+- `table_name`: The name of the remote table.
+- `boolean_expression`: The filter condition.
+- `PROPERTIES`: Currently, only the `verbose` property is supported. It is used to return detailed warmup metrics.
+
+CACHE_SELECT is a synchronous process and it can warm up only one table at a time. Upon successful execution, it will return cache-related metrics.
+
+### Warm up all data in the remote table
```
Comment (EsoragotoSpirit) [inline docs/en/data_source/data_cache_warmup.md:34]: remote? external 是不是好点？

File: docs/en/data_source/data_cache_warmup.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+| TaskName     | Status    |
++--------------+-----------+
+| always_cache | SUBMITTED |
++--------------+-----------+
+1 row in set (0.03 sec)
+```
+
+### Manage CACHE SELECT tasks
+
+#### Query created tasks
```
Comment (EsoragotoSpirit) [inline docs/en/data_source/data_cache_warmup.md:108]: suggestion: query --> view

File: docs/en/data_source/data_cache_warmup.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+#### Drop tasks
+
+```sql
+DROP TASK <task_name>
+```
+
+## Use cases
+
+1. During PoC performance testing, if you want to assess StarRocks' performance without interference from external storage systems, you can use the CACHE_SELECT statement to load the data of the table to test into the data cache in advance.
```
Comment (EsoragotoSpirit) [inline docs/en/data_source/data_cache_warmup.md:143]: CACHE_SELECT 下划线

Comment (Smith-Cruise) [reply docs/en/data_source/data_cache_warmup.md:143]: `CACHE SELECT` 是没有下划线的

File: docs/zh/data_source/data_cache_warmup.md (hunk 1 above)
Comment (evelynzhaojie) [reply docs/zh/data_source/data_cache_warmup.md]: added

File: docs/zh/data_source/data_cache_warmup.md (hunk 2 above)
Comment (evelynzhaojie) [reply docs/zh/data_source/data_cache_warmup.md]: simo 说需要讨论一下，应该是需要加，等 QA 测试完再确定。

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Pull request merged**